src/main/webapp/i18n/en/exam.json

```diff
@@ -209,7 +209,7 @@
                 "question": "Are you sure you want to change the submission state?",
                 "submitText": "This will set the submission date of the exam to this moment, and set the Submitted status to 'Yes'. Un-submitting the student exam later is possible.\n",
                 "unSubmitText": "This will delete the submission date and set the Submitted status to 'No'.",
```

Comment:
maybe we should either use "submission state" or "Submitted status", I think it is both referring to the value of the variable `submitted` being true (for submitted) or false (for unsubmitted)

Reply:
Could you make a concrete change suggestion so I can better see what you mean?

Reply:
sorry, that was a bit vague :) here the suggestions:
```
"submitText": "This will set the submission date of the exam to this moment, and the submission state to 'submitted'. Un-submitting the student exam later is possible.\n",
"unSubmitText": "This will delete the submission date and set the submission state to 'unsubmitted'.",
```
Would you say "set to this moment" should be replaced by "set to this point in time"? I just thought about that.

Reply:
Those changes look good to me!
And I have no opinion one way or the other on those two options, I think they're equally good/clear